spec/factories/plant_trial.rb

```diff
@@ -1,6 +1,6 @@
 FactoryGirl.define do
   factory :plant_trial do
-    sequence(:plant_trial_name) {|n| "#{Faker::Lorem.characters(5)}_#{n}"}
+    sequence(:plant_trial_name) {|n| "#{Faker::Lorem.characters(20)}_#{n}"}
```

Comment:
Again, I believe you don't have to do this, if you use "n". Apart from the case that you need it to be 20+ characters for some other reason.